src/components/header/styles.js

```diff
@@ -168,8 +172,19 @@ export const HeaderWrapper = styled.div`
         background-color: transparent;
       }
 
+      &:hover {
+        color: ${darken(0.2, theme.colors.darkGrey)};
+        &:not([type='button']):after {
+          content: '';
+          background-color: ${theme.colors.green};
+        }
+        svg {
+          fill: ${darken(0.2, theme.colors.darkGrey)};
+        }
+      }
+
       &.active {
-        &::after {
+        &:not([type='button']):after {
```

Comment:
This will break showing 'topics' as active when a child route is selected. We need a more specific solution.